lib/manager/actions/projects.js

```diff
@@ -37,7 +37,7 @@ const runningFetchFeedsForProject = createVoidPayloadAction('RUNNING_FETCH_FEED_
 // Used by component
 export const createProject = createAction(
   'CREATE_PROJECT',
-  (payload: Project) => payload
+  (payload: $Shape<Project> & { organizationId: string }) => payload
```

Comment:
Is an organizationId always required? I thought this was optional, but maybe I'm mistaken.